package.json

```
@@ -17,7 +17,8 @@
 	"scripts": {
 		"build": "tsc --project tsconfig.production.json",
 		"build:all": "tsc",
-		"debug": "concurrently \"tsc -w\" \"nodemon\"",
+		"debug": "nodemon --exec \"tsc && node --inspect=5859 -r tsconfig-paths/register -r ts-node/register build/src/app.js\"",
+		"debug-ts": "nodemon --exec \"node --inspect=5859 -r tsconfig-paths/register -r ts-node/register src/app.ts\"",
```

```suggestion
		"debug-ts": "nodemon --exec \"ts-node --inspect=5859 -r tsconfig-paths/register -r ts-node/register src/app.ts\"",
```
shouldnt be this node-ts?

---

I'm pretty sure this is ts-node:
https://github.com/TypeStrong/ts-node
Works as expected. Or am I missing something here...?